tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -673,15 +736,29 @@ public TestSetup apply(Long subscribers) throws Throwable {
           // sub1 now has 18 pending
           // sub2 has 0 pending
 
-          final T z = sendNextTFromUpstream();
-          expectNextElement(sub1, z);
-          sub2.expectNone(); // since sub2 hasn't requested anything yet
+          if (doesCoordinatedEmission()) {
+            sub2.expectNone(); // since sub2 hasn't requested anything yet
 
-          sub2.request(1);
-          expectNextElement(sub2, z);
+            sub2.request(1);
 
-          if (totalRequests == 3) {
-            expectRequest();
+            final T z = sendNextTFromUpstream();
+            expectNextElement(sub1, z);
+            expectNextElement(sub2, z);
+
+            if (totalRequests == 3) {
```

Comment:
Since this is the same behavior on each side of the branch, perhaps put it after the branch?